data_store/tests/test_pydicom.py

```diff
@@ -0,0 +1,25 @@
+"""Tests for the interface with pydicom."""
+
+import os
+from pathlib import Path
+
+import pytest
+
+from pydicom.data import get_testdata_file
+
+
+class TestPydicom:
+    """Test the interface with pydicom works correctly."""
+    def setup(self):
+        cur_dir = os.path.dirname(os.path.realpath(__file__))
+        self.data_path = Path(cur_dir).joinpath("..", "..", "data").resolve()
+
```

Comment:
can simplify similar to other pathlib ...

Reply:
Done